runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcDirectGetWorkStream.java

```diff
@@ -99,18 +101,18 @@ private GrpcDirectGetWorkStream(
       int logEveryNStreamFailures,
       ThrottleTimer getWorkThrottleTimer,
       Supplier<GetDataStream> getDataStream,
-      Supplier<CommitWorkStream> commitWorkStream,
-      WorkItemProcessor workItemProcessorFn) {
+      Supplier<WorkCommitter> workCommitter,
+      WorkItemScheduler workItemSchedulerFn) {
```

Comment:
nit: drop Fn from param name and member variable since it's a class not only a function

Reply:
done